src/darwin/Framework/CHIP/MTRDeviceController_XPC.mm

```diff
@@ -467,7 +467,7 @@ - (oneway void)controller:(NSUUID *)controller controllerConfigurationUpdated:(N
 
     NSArray * deviceInfoList = MTR_SAFE_CAST(configuration[MTRDeviceControllerRegistrationNodeIDsKey], NSArray);
 
-    MTR_LOG("Received controllerConfigurationUpdated: controllerNode ID %@ deviceInfoList %@", self.controllerNodeID, deviceInfoList);
+    MTR_LOG("Received controllerConfigurationUpdated: controllerNodeID %@ compressedFabricID %016llX deviceInfoList %@", self.controllerNodeID, self.compressedFabricID.unsignedLongLongValue, deviceInfoList);
```

Comment:
```suggestion
    MTR_LOG("Received controllerConfigurationUpdated: controllerNodeID %@ compressedFabricID %016lluX deviceInfoList %@", self.controllerNodeID, self.compressedFabricID.unsignedLongLongValue, deviceInfoList);
```
since it's unsigned?